docs/topics/pages.md

```diff
@@ -335,9 +335,10 @@ Wagtail allows the nesting of other models within a page. This is useful for cre
 
 Each inline model requires the following:
 
--   It must inherit from {class}`wagtail.models.Orderable`
 -   It must have a `ParentalKey` to the parent model
```

Comment:
I guess there's not much need for this to be a bullet point, now that there's only one of them :-) This could change to something like:

```
An inline model must have a `ParentalKey` pointing to the parent model. It can also inherit from {class}`wagtail.models.Orderable` to allow reordering of items in the admin interface.
```

(then continue with the note about django-modelcluster)